sources/input/input.c

```diff
@@ -1,15 +1,15 @@
 /* ************************************************************************** */
 /*                                                                            */
-/*   input.c - map keypresses to editor functions                             */
+/*                                                        :::      ::::::::   */
+/*   input.c                                            :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: alberrod <alberrod@student.42urduliz.co    +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/01/01 00:00:00 by ed                #+#    #+#             */
+/*   Updated: 2024/07/17 23:32:46 by alberrod         ###   ########.fr       */
```

Comment:
```suggestion
/*   input.c - map keypresses to editor functions                             */
/*                                                                            */
/*   Created: 2023/11/27 18:32:33 by cezelot                                  */
/*   Updated: 2024/07/17 23:32:46 by alberrod                                 */
/*                                                                            */
/*   Copyright (C) 2024 Ismael B. Hamed, Alberto Rodriguez                    */
/*                                                                            */
```

Reply:
The license notice must not change other than updating the following information: the date of modification, the author of the modification and the copyright notice.